samples/places-autocomplete-addressform/src/index.njk

```diff
@@ -13,53 +13,46 @@
+https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform
     -->
 
-<table id="address">
-  <tr>
-    <td class="label">Street address</td>
-    <td class="slimField">
-      <input class="field" id="street_number" disabled="true" />
-    </td>
-    <td class="wideField" colspan="2">
-      <input class="field" id="route" disabled="true" />
-    </td>
-  </tr>
-  <tr>
-    <td class="label">City</td>
-    <td class="wideField" colspan="3">
-      <input class="field" id="locality" disabled="true" />
-    </td>
-  </tr>
-  <tr>
-    <td class="label">State</td>
-    <td class="slimField">
-      <input class="field" id="administrative_area_level_1" disabled="true" />
-    </td>
-    <td class="label">Zip code</td>
-    <td class="wideField">
-      <input class="field" id="postal_code" disabled="true" />
-    </td>
-  </tr>
-  <tr>
-    <td class="label">Country</td>
-    <td class="wideField" colspan="3">
-      <input class="field" id="country" disabled="true" />
-    </td>
-  </tr>
-</table>
+<form action="" method="get" autocomplete="off">
+  <span class="title">Sample address form for North America</span>
+  <label class="fullfield" for="gmp-a1">
+    <!-- Avoid the word "address" in id, name, or label text to avoid browser autofill from conflicting with Place Autocomplete. Star or comment bug https://crbug.com/587466 to request Chromium to honor autocomplete="off" attribute. -->
+    <span class="formlabel" id="a1-label">Deliver to*</span>
+    <input type="text" id="gmp-a1" name="gmp-a1" aria-labelledby="a1-label" required autocomplete="off">
+  </label>
+  <label class="fullfield" for="address2">
+    <span class="formlabel" id="address2-label">Apartment, suite, etc.</span>
+    <input type="text" id="address2" name="address2" aria-labelledby="address2-label">
+  </label>
+  <label class="fullfield" for="locality">
+    <span class="formlabel" id="locality-label">City*</span>
+    <input type="text" id="locality" name="locality" aria-labelledby="locality-label" required>
+  </label>
 
-{% endblock %}
+  <label class="slimfield-left" for="administrative_area_level_1">
+    <span class="formlabel" id="administrative_area_level_1-label">State/Province*</span>
+    <input type="text" id="administrative_area_level_1" name="administrative_area_level_1"
+      aria-labelledby="administrative_area_level_1-label">
+  </label>
+  <label class="slimfield-right" for="postal_code">
+    <span class="formlabel" id="postal_code-label">Postal code*</span>
+    <input type="text" id="postal_code" name="postal_code" aria-labelledby="postal_code-label">
+  </label>
+
+  <label class="fullfield" for="country">
+    <span class="formlabel" id="country-label">Country/Region*</span>
+    <input type="text" id="country" name="country" aria-labelledby="country-label">
+  </label>
```

Comment:
great to see a11y labels! however, the wrapping <label> element is all you need. if "*" should mean required (is that noted somewhere?) you also want the `required` attribute

```
<label class="fullfield">
     <span class="formlabel">Country/Region*</span>
     <input name="country" required>
   </label>
```

Reply:
done